eZ/Publish/Core/Base/Exceptions/ContentFieldValidationException.php

```diff
@@ -53,4 +64,32 @@ public function getFieldErrors()
     {
         return $this->errors;
     }
+
+    private function generateValidationErrorsMessages(): string
+    {
+        $validationErrors = $this->collectValidationErrorsMessages();
```

Comment:
IMHO we should limit errors included into message to some reasonable number e.g. 32 and add text ellipsis if this limit is exceeded.